### ISSUE TITLE
about training time

### ISSUE THREAD

**Issue by wannanfeng** [HKUDS/RecGPT#1]
Good work, but I have a question about the training time. In your paper, you wrote: "We implement early stopping with 10-epoch patience to prevent overfitting." However, I trained according to your instructions, using 8x80GB GPUs and a batch size of 80. However, after using tqdm, I observed that it takes 10 hours to complete one epoch. In other words, if I train for 10 epochs and find the best ckpt, it will take many days.

**Comment by yangqin-jiang** [comment]
Hi,

Thanks for your attention. In fact, the model's loss on the validation set has already begun to converge by the second epoch or the third epoch. With 8x80GB GPUs, one or two days is sufficient to complete the training. 

Sincerely,
Yangqin

**Comment by yangqin-jiang** [comment]
Due to a period of inactivity, this issue will be closed for now. Feel free to continue asking questions!
